.idea/compiler.xml

```diff
@@ -18,5 +18,15 @@
         <processorPath useClasspath="true" />
       </profile>
     </annotationProcessing>
+    <bytecodeTargetLevel>
```

Comment:
sync.

On Wed, Mar 23, 2016 at 5:14 PM, Appu Goundan <notifications@github.com>
wrote:

> In .idea/compiler.xml
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/529#discussion_r57238416>
> :
>
> > @@ -18,5 +18,15 @@
> >          <processorPath useClasspath="true" />
> >        </profile>
> >      </annotationProcessing>
> > +    <bytecodeTargetLevel>
>
> What triggered it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/529/files/461c64498400884b572110640440748b5b8422ec#r57238416>
>